src/actionSelector.ts

```diff
@@ -118,7 +118,7 @@ export class ActionSelector {
         loggedAction => loggedAction.href === action.href
       )
     ) {
-      log(`Cannot return action twice: ${JSON.stringify(action)}!`);
+      logger.debug(`Cannot return action twice: ${JSON.stringify(action)}!`);
```

Comment:
🙃 Program flow logging. 

```suggestion
      logger.trace(`Cannot return action twice: ${JSON.stringify(action)}!`);
```

Reply:
see comment above :)